apps/parsley/src/utils/string/index.ts

```diff
@@ -1,10 +1,10 @@
-/**
- * `copyToClipboard` copies text to a user's clipboard.
- * @param textToCopy - text to be copied
- */
-export const copyToClipboard = async (textToCopy: string) => {
-  await navigator.clipboard.writeText(textToCopy);
-};
+import {
+  copyToClipboard,
+  shortenGithash,
+  trimStringFromMiddle,
+} from "@evg-ui/lib/utils/string";
+
+export { shortenGithash, trimStringFromMiddle, copyToClipboard };
```

Comment:
We should not be rexporting these values from this file instead we should just import the funcitons directly from "@evg-ui/lib/utils/string" when needed